### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Diggsey** [christolliday/limn#35]
It's probably way too early to actually take any action here, but one of the common criticisms with applications that do custom UI rendering is that they typically have terrible accessibility, and do not follow the standards for how a GUI should behave.

I think it would be good to at least set a goal of having accessibility on-par with applications written to use the platform-native controls.

In the long run, this might mean integrating with [the windows automation API](https://msdn.microsoft.com/en-us/library/windows/desktop/ee671698(v=vs.85).aspx) and similar on other platforms.

In the short term, it means taking settings from the OS (double click speed, key repeat delay, etc.) implementing keyboard navigation and mouse event capturing to mirror "normal" UI behaviour.

I'm really excited by this project! It seems like a great combination (webrender/cassowary/rust) and rust really needs good and easy to use GUI libraries.
